dbms/src/TableFunctions/TableFunctionRandom.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <TableFunctions/ITableFunction.h>
+
+namespace DB
+{
+/* random(structure, limit) - creates a temporary storage filling columns with random data
+ * random is case-insensitive table function
```

Comment:
Case insensitive makes sense only if the function corresponds to the function from SQL standard or some other DBMS and we make it case insensitive for compatibility.

Reply:
OK